ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTPayload.scala

```diff
@@ -158,14 +182,35 @@ object AuthServiceJWTCodec {
       val scopes = scope.toList.collect({ case JsString(scope) => scope.split(" ") }).flatten
       // We're using this rather restrictive test to ensure we continue parsing all legacy sandbox tokens that
       // are in use before the 2.0 release; and thereby maintain full backwards compatibility.
-      if (scopes.contains(scopeLedgerApiFull))
+      val audienceValue = readOptionalStringOrSingletonArray(propAud, fields)
+      val participantIdForParticipantIdFormat = {
+        // Additionally to the tokens with scope containing `daml_ledger_api`, we support tokens
+        // with the audience which starts with `https://daml.com/participant/jwt/aud/participant/${participantId}`
+        // where `${participantId}` is mandatory
+        // As required for JWTs, additional fields can be in a token but will be ignored (including scope)
+        audienceValue match {
+          case Some(s) if s.startsWith(audPrefix) =>
+            Some(s.substring(audPrefix.length)).filter(_.nonEmpty)
+          case _ =>
+            None
+        }
+      }
+      if (scopes.contains(scopeLedgerApiFull)) {
         // Standard JWT payload
         StandardJWTPayload(
-          participantId = readOptionalString("aud", fields),
+          participantId = audienceValue,
           userId = readOptionalString("sub", fields).get, // guarded by if-clause above
           exp = readInstant("exp", fields),
+          format = StandardJWTTokenFormat.Scope,
         )
-      else {
+      } else if (participantIdForParticipantIdFormat.isDefined) {
+        StandardJWTPayload(
+          participantId = participantIdForParticipantIdFormat,
+          userId = readOptionalString("sub", fields).get, // guarded by if-clause above
+          exp = readInstant("exp", fields),
+          format = StandardJWTTokenFormat.ParticipantId,
+        )
+      } else {
         if (scope.nonEmpty)
           logger.warn(
             s"Access token with unknown scope \"${scope.get}\" is being parsed as a custom claims token. Issue tokens with adjusted or no scope to get rid of this warning."
```

Comment:
This warning deals with unknown scopes. 
Should there also be a warning for unknown/invalid aud values?